unity-environment/ProjectSettings/TagManager.asset

```diff
@@ -22,6 +22,7 @@ TagManager:
   - orangeBlock
   - block
   - orangeGoal
+  - crawlerTarget
```

Comment:
I believe it is better to keep these tags as generic as possible as they are shared across all scenes.

Reply:
changed to "target"